### ISSUE TITLE
同样是一张3090，m3蒸馏版本，只得到了论文宣称的一半吞吐量

### ISSUE THREAD

**Issue by ztxdcyy** [mlvlab/EfficientViM#4]
[2025-04-05 13:25:39 EfficientViM_M3](main.py 394): INFO Test: [130/196]        Time 0.038 (0.393)      Loss 0.5628 (0.7906)    Acc@1 85.938 (80.561)   Acc@5 97.266 (94.844)   Mem 16848MB

batchsize = 256 256/0.038 大约6600FPS，论文宣称在一张3090上可以得到11000FPS吞吐量。

Table5提到蒸馏版本也不会降低吞吐量，甚至会提高一些准确率：
<img width="439" alt="Image" src="https://github.com/user-attachments/assets/7dc92891-eff2-4164-ac08-010e078b071b" />

> “To validate the effectiveness of EfficientViM in speed-accuracy trade-offs, we measure the throughput (im/s), and latency (ms) with the batch size of 256 on an NVIDIA RTX 3090 along with the accuracy, and present the results in Table 3. EfficientViM outperforms all previous efficient networks in both speed and accuracy.” 
> 为了验证 EfficientViM 在速度-精度权衡方面的有效性，我们在 NVIDIA RTX 3090 上测量了批量大小为 256 的吞吐量（ im/s ）和延迟（毫秒）以及准确性，并将结果显示在表 3 中。EfficientViM 在速度和准确性方面都优于以前的所有 Efficient 网络。

**Comment by LSanghyeok** [comment]
Hello! 
For throughput in the paper, we directly adopted the `speed_test.py` in EfficientViT and SHViT. All throughput measurements were conducted consistently using these standalone scripts rather than the training pipelines, ensuring a fair comparison across models. 
You can review the exact scripts for reporting throughput here:
SHViT: https://github.com/ysj9909/SHViT/blob/main/speed_test.py
EfficientViT https://github.com/microsoft/Cream/blob/main/EfficientViT/classification/speed_test.py

**Comment by ztxdcyy** [comment]
thank you for your reply. i will check it later

**Comment by ztxdcyy** [comment]
![Image](https://github.com/user-attachments/assets/80578473-ee88-487f-a1c8-dc9601d5969e)

sorry for my rude, thank you very much!

Here is my practice:
1. import的模型改掉
2. model的n 循环的时候改掉
3. 下载utils，需要再下载apex库，因为其中的一个替换norm的高性能实现需要用到apex库
4. python speed_test.py --pretrained /path/to/m3-dist.pth
